conanfile.py

```diff
@@ -57,14 +57,16 @@ class Morpheus(ConanFile):
         "fPIC": [True, False],
         "tools": [True, False],
         "build_docs": [True, False],
-        "link_with_mold": [True, False]
+        "link_with_mold": [True, False],
```

Comment:
Let's recorder all of the setting to be in alphabetical order (not so necessary when there is 3-4 but I think we are going to end up with lots so it will make searching for a specific entry easier).